plugins/inputs/dmcache/dmcache_linux_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !linux
```

Comment:
This doesn't seem right, why are you removing test on linux and calling it dmcache_linux_test?